plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -395,7 +395,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
 				// Allow all API calls to the nodes
 				rbacv1helpers.NewRule("proxy").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
-				rbacv1helpers.NewRule("*").Groups(legacyGroup).Resources("nodes/proxy", "nodes/metrics", "nodes/stats", "nodes/log").RuleOrDie(),
+				rbacv1helpers.NewRule("*").Groups(legacyGroup).Resources("nodes/proxy", "nodes/metrics", "nodes/stats", "nodes/log", "nodes/checkpoint").RuleOrDie(),
```

Comment:
We need to have nodes/checkpoint guarded under a feature gate.

Reply:
hmm I thought about that but could not see a downside of having this permission even if the feature gate is not enabled other than its not least privilege.

Reply:
aah never mind I see we do that for other features and why not follow the least privilege principle. Fixed. Thanks!